cmd/snap/cmd_info_test.go

```diff
@@ -306,8 +308,8 @@ description: |
   https://snapcraft.io/
 snap-id:   mVyGrEwiqSi5PugCwyH7WgpoQLemtTd6
 tracking:  beta
-refreshed: 0001-01-01T00:00:00Z
-installed: 2.10 (1) 1kB disabled
+refreshed: 2006-01-02T22:04:07Z
+current: 2.10 (1) 1kB disabled
```

Comment:
Same.

... and we seem to be lacking a case that includes the channel map, which would justify the different alignment.